command/harness/destroy.go

```diff
@@ -45,6 +46,7 @@ func HandleDestroy(ctx context.Context, r *VMCleanupRequest, s store.StageOwnerS
 			logrus.WithError(err).
 				WithField("retry_count", cnt).
 				WithField("stage_runtime_id", r.StageRuntimeID).
+				WithField("task_id", r.TaskID).
```

Comment:
Do we pass in task ID from delegate now? How about in the dlite flow?

Reply:
task id is passed as correlation id in request.